### PR TITLE
Aluminum Chonk, Semiautomatic, 9mm statfixes

### PR DESCRIPTION
**Pull request by tenmillimaster**
#### Summary
```SUMMARY: Content "Updates M9"```

#### Purpose of change
An m9 in game weighed as much as a glock. The M9 is no longer popular.

#### Describe the solution
Updated name, description, and weight. Performed volume analysis on M9A1 image. Description set to reflect some 'then and now' background info. Weight set to real life value.

#### Describe alternatives you've considered
Watching Die Hard.

#### Testing
Loaded game.

#### Additional context
None.

### CODE REVIEW

File: data/json/items/gun/9mm.json [hunk 1]
```diff
@@ -344,14 +344,14 @@
     "copy-from": "pistol_base",
     "looks_like": "glock_17",
     "type": "GUN",
-    "name": "Beretta M9",
-    "description": "A very popular 9x19mm pistol, the M9 has been the standard issue sidearm of the US army since 1985.",
-    "weight": "590 g",
-    "volume": "500 ml",
+    "name": "Beretta M9A1",
+    "description": "An iconic 9x19mm pistol, the M9 was the standard issue sidearm of US armed force from 1985 only until recently.  It was formerly popular amongst law enforcement before the advent of commercially available polymer framed handguns and the FBI's adoption of .40S&W.",
```
Comment (Night-Pryanik) [inline data/json/items/gun/9mm.json]:
```suggestion
    "description": "An iconic 9x19mm pistol, the M9 was the standard issue sidearm of US armed force from 1985 only until recently.  It was formerly popular amongst law enforcement before the advent of commercially available polymer-framed handguns and the FBI's adoption of .40S&W.",
```

**Comment by Night-Pryanik** [comment]
Why not M9A3 variant?

**Comment by tenmillimaster** [comment]
> Why not M9A3 variant?

It wasn't adopted.

**Comment by LaVeyanFiend** [comment]
What pictures did you use to ImageJ the volume from?

**Comment by LaVeyanFiend** [comment]
Imma just put the source for the weight and measurements for ImageJ here:
http://www.beretta.com/en-us/m9a1/

**Comment by LaVeyanFiend** [comment]
Duplicated the ImageJ of this from this photo:
image
Got these results(square mm) using the overall height of 137mm:
image
What did you use for width? I used the 1.3"/33.02mm grip width and got 461ml 😕

**Comment by LaVeyanFiend** [comment]
Replicated with photo from Beretta, using overall length of 216(height over official by 4mm):
http://www.beretta.com/assets/0/15/DimGalleryLarge/m9a1_zoom002.jpg
image
Got 490ml from using grip width.

**Comment by tenmillimaster** [comment]
38mm, same source. I just redid my calc based on your photo, setting the reference length to the OAL.

The difference is we're using different measurements from beretta, OAL width and grip width.

I'm going to average the two widths, and use 35.5mm.


Thank you so much for catching this.

**Comment by Outta-my-way-burrocrat** [comment]
Does it need to have the same durability as the Glocks?

**Comment by tenmillimaster** [comment]
> Does it need to have the same durability as the Glocks?

It hasn't been changed since 0.B, and is a bit of an underestimation of the system, so no, it shouldn't.

**Pull request merged**